yellowbrick/classifier/confusion_matrix.py

```diff
@@ -103,6 +108,20 @@ def score(self, X, y, sample_weight=None, percent=True):
         """
         y_pred = self.predict(X)
 
+
+        if self.label_mapping:
+            # if a mapping is passed to class apply it here.
+            y = character_mapping(y, self.label_mapping)
+            y_pred = character_mapping(y_pred, self.label_mapping)
```

Comment:
This makes sense, and I like it! It's a good solution, if the user is able to supply a mapping (see above).